incubator/scylladb/README.md

```diff
@@ -0,0 +1,140 @@
+# ScyllaDB Helm Chart on Google Kubernetes Engine (GKE)
+
+ ## What is Helm Charts? Why use a Helm Chart? 
```

Comment:
Don't think we need to add this section. I would expect the user to know what a Helm chart is if they get here.

Reply:
Yes, for now. Mainly because of storageclass. I run this locally also but I have to create the storageclass and pvs manually. For AWS and Azure I would have to include different storageclasses. If you have an idea on how to solve this problem, I would gladly make it more generic. But for now, I'd rather just support GKE and add support to other clouds or deployments in a paced manner.